benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherSelectCandidatesAzureBenchmark.cs

```diff
@@ -44,7 +44,7 @@ public void Baseline()
 
                 var candidates = _baseline.Matchers[sample].SelectCandidates(path, segments);
 
-                var endpoint = candidates.Candidates[0].Endpoint;
+                var endpoint = candidates[0].Endpoint;
```

Comment:
This is the first of a few changes that spread out all over teh codez. `CandidateSet` is now public since we did away with groups. 

So, a whole bunch of things now work with (immutable) `Candidate[]` where the used to work with `CandidateSet`, because candidate set is now mutable and public.

Reply:
So everwhere you see `.Candidates` deleted from the code, that's the reason.